src/tests/functional/plugin/shared/src/low_precision_transformations/concat_with_neighbors_graph_transformation.cpp

```diff
@@ -43,6 +45,19 @@ void ConcatWithNeighborsGraphTransformation::SetUp() {
         "");
```

Comment:
Please add convolution as an additional layer
```suggestion
        "convolution");
```

Reply:
Applied